lib/warewulf/bmc/ipmitool.tmpl

```diff
@@ -1,6 +1,6 @@
 {{ $cmd := "ipmitool" }}
 {{ if .Interface }}{{ $cmd = cat $cmd "-I" .Interface }}{{ end }}
-{{ if .EscapeChar }}{{ $cmd = cat $cmd "-e" .EscapeChar }}{{ end }}
+{{ if .EscapeChar }}{{ $cmd = cat $cmd "-e" (printf "'%s'" .EscapeChar) }}{{ end }}
```

Comment:
Can you either use double quotes, or update `.UserName` and `.Password` below to use single quotes? (I don't think I have much of an opinion either way, but I just think they should match.)

Reply:
I'll switch to double, they also get the job done.